src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -436,6 +438,24 @@ unsigned uct_rc_verbs_iface_post_recv_always(uct_rc_verbs_iface_t *iface, unsign
     return count;
 }
 
+static void uct_rc_verbs_iface_vfs_refresh(uct_iface_h tl_iface)
+{
+    uct_rc_verbs_iface_t *iface = ucs_derived_of(tl_iface, uct_rc_verbs_iface_t);
+    uct_rc_verbs_ep_t *ep;
+
+    /* Add iface resources */
+    uct_rc_iface_vfs_populate(&iface->super);
+
+    /* Add objects for EPs */
+    ucs_list_for_each(ep, &iface->super.ep_list, super.list) {
+        ucs_vfs_obj_add_dir(iface, ep, "ep/%p", ep);
+        ucs_vfs_obj_add_ro_file(ep, ucs_vfs_show_primitive,
+                                &ep->qp->qp_num, UCS_VFS_TYPE_U32_HEX,
```

Comment:
Why do we need this value in hex format?

Reply:
it was aligned with MLX5 transports:
https://github.com/openucx/ucx/blob/f48d5e2558162a7f8633df158905c104d2b06327/src/uct/ib/mlx5/ib_mlx5.c#L494

Reply:
Guess it can be `UCS_VFS_TYPE_U32` instead in both places.
Maybe @yosefe can clarify this, as the author of original code.

Reply:
better print QPs as HEX since it's how FW/HW debug tools print them

Reply:
Ok, thanks.